gensim/test/test_summarization.py

```diff
@@ -128,3 +128,14 @@ def test_corpus_summarization_ratio(self):
             expected_summary_length = int(len(corpus) * ratio)
 
             self.assertEqual(len(selected_docs), expected_summary_length)
+
+    def test_keywords_runs(self):
+        pre_path = os.path.join(os.path.dirname(__file__), 'test_data')
+
+        with utils.smart_open(os.path.join(pre_path, "mihalcea_tarau.txt"), mode="r") as f:
+            text = f.read()
+
+        kwds = keywords(text)
```

Comment:
Can you try the same thing, but also using `keywords(to_unicode(text))`?

That way we test that both utf8 input and unicode input work.